pkg/config/generate.go

```diff
@@ -181,7 +181,10 @@ func Generate(m *api.Manifest) (c *Config, err error) {
 			cn: "servicecatalog-api",
 			dnsNames: []string{
 				"servicecatalog-api",
-				"apiserver.kube-service-catalog.svc", // TODO: unclear how safe this is
+				"servicecatalog-api." + m.ResourceGroup,
+				"servicecatalog-api." + m.ResourceGroup + ".svc",
+				"servicecatalog-api." + m.ResourceGroup + ".svc.cluster.local",
```

Comment:
`"servicecatalog-api." + m.ResourceGroup + ".svc.cluster.local"` is the only one needed here, the others can be removed.

Reply:
The core api server is failing if `"apiserver.kube-service-catalog.svc"` is missing, that's why it's added here.

Reply:
The rest servicecatalog-api hostnames can probably be removed.